File: CrossTemplate.VB/CrossTemplate/CrossTemplate.vb
Imports Xamarin.Forms

Public Class App
    Inherits Application

    Public Sub New()
        Dim stack = New StackLayout With {.VerticalOptions = LayoutOptions.Center}

        stack.Children.Add(
            New Label With {.HorizontalTextAlignment = TextAlignment.Center,
                            .Text = "Welcome to Xamarin.Forms with Visual Basic.NET"}
        )

        MainPage = New ContentPage With {.Content = stack}
    End Sub

End Class

File: CrossTemplate.FSharp/CrossTemplate/CrossTemplate.fs
namespace CrossTemplate

open Xamarin.Forms

type App() as this =
    inherit Application()

    let stack = new StackLayout(VerticalOptions = LayoutOptions.Center)
    do
        stack.Children.Add(
            new Label(HorizontalTextAlignment = TextAlignment.Center,
                      Text = "Welcome to Xamarin.Forms with F#")
        )

        this.MainPage <- new ContentPage(Content = stack)
